highlight/events-explainer.md

```diff
@@ -27,7 +27,7 @@ interface HighlightPointerEvent : PointerEvent {
 }
 ```
 
-For the selected highlight/range pair, a HighlightPointerEvent object must be dispatched with the relevant range as its `range` and the Highlight as its `currentTarget`. The event's `target` will be set to the element the user interacted with.
+For the selected highlight/range pair, a HighlightPointerEvent object must be dispatched with the relevant range as its `range` and the Highlight as its `currentTarget`. In order to retain web compatibility and current PointerEvents' behaviour, the event's `target` will remian being the element the user interacted with throughout the event's propagation.

```

Comment:
```suggestion
When a user performs a pointer action on part of an element containing a highlighted `Range`, a `HighlightPointerEvent` object must be fired on the `Highlight` that that `Range` belongs to. `HighlightPointerEvent.range` will be set to that `Range` so that the event listener can determine which one of the `Highlight`'s ranges the event applies to. Propagation for the event will work as if the element was the first DOM parent of the `Highlight` (see below examples).

One approach would be to set `HighlightPointerEvent.target` to the `Highlight`. But, this poses a web compatibility problem. Currently a `PointerEvent`'s `target` can only be an element, so `PointerEvent` handlers will expect this and encounter problems if they receive a `HighlightPointerEvent` whose `target` is a `Highlight`.

So in order to retain web compatibility and not break `PointerEvent` handlers in existing code, the event's `target` will be set to the element rather than the `Highlight`.

```
I'm trying to be really clear about why we're making the event behave in this way. Does the above explanation seem ok to you?